app/javascript/components/student/editor/FileEditor.tsx

```diff
@@ -53,9 +56,9 @@ export const FileEditor = forwardRef<FileEditorHandle, FileEditorProps>(
           editorDidMount={editorDidMount}
           onRunTests={onRunTests}
           options={options}
-          value={file.content}
-          theme={'vs'}
-          keybindings={Keybindings.DEFAULT}
+          defaultValue={file.content}
```

Comment:
I changed this to `defaultValue` as the value gets reset when we change the theme.